index.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <div>
+        <nav id="navbar">
+            <ul class="navlinks">
+                <li>Home</li>
+                <li>Projects</li>
+                <li>About</li>
+            </ul>
+        </nav>
+        <div id="welcome-section">
+            <h1>Abu bakar Siddique</h1>
+        </div>
+
+        <div id="projects">
+            <div id="projects-title">
+                <h1>Projects</h1>
+            </div>
+            <div class="flex-container">
+                <div class="project-tile">
+                    <h4>Javascript Calculator</h4>
+                    <div class="image"></div>
+
+                    <a href="https://codepen.io/freeCodeCamp/full/zNBOYG">Click Here to go to view project</a>
+
+                </div>
+                <div class="project-tile">
+                    <h4>Tictactoe Game</h4>
+                    <div class="image"></div>
+                    <a href="https://codepen.io/freeCodeCamp/full/zNBOYG">Click Here to go to view project</a>
+
+                </div>
+                <div class="project-tile">
+                    <h4>Tetris</h4>
+                    <div class="image"></div>
+                    <a href="https://codepen.io/freeCodeCamp/full/zNBOYG">Click Here to go to view project</a>
+
+                </div>
+            </div>
+        </div>
+    </div>
+    <div>
+        <div id="footer-title">
+            <h2>Let's Work Together...</h2>
+        </div>
```

Comment:
It's my pleasure to check your code I will suggest you here <footer><footer/> will be the best. 🥇

Reply:
- [ ] This comment is not clear and it doesn't convey any message to the author of the project. This makes it difficult to understand what to do next. Kindly make the statement clearer and more understandable. 👍 

### Example of a good comment that is clear to understand

- *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*

### Example of a bad comment that is not easily understandable
- *The README is not descriptive, why don't you make it better?*

#### Hint: Kindly edit your initial statement and make your comment clear and easy to understand. 💻

Reply:
@Gambit142 I got your point and edit my commits.